src/ir_Argo.cpp

```diff
@@ -0,0 +1,282 @@
+/*
+Node MCU/ESP8266 Sketch to emulate Argo Ulisse 13 DCI remote
```

Comment:
Can you please also add what models the remote/code is known to control. Just the one you have access to and tested against is fine for starters.